src/emc/kinematics/genhexkins.c

```diff
@@ -11,7 +11,7 @@
 *    
 * Copyright (c) 2004 All rights reserved.
 *
-* Last change: 2014.12.22
+* Last change: 2016.08.04
```

Comment:
Agree. Last change date is removed.
Not sure about the copyright date, it was in the initial genhexkins.... so I leave it there.